### ISSUE TITLE
Emojis not recognised in Windows 10

### ISSUE THREAD

**Issue by spences10** [zeit/hyper#3113]
- [x] I am on the [latest](https://github.com/zeit/hyper/releases/latest) Hyper.app version
- [x] I have searched the [issues](https://github.com/zeit/hyper/issues) of this repo and believe that this is not a duplicate

- **OS version and name**: Windows 10 build 17704
- **Hyper.app version**: Canary 2.1.0-canary2
- **Link of a [Gist](https://gist.github.com/) with the contents of your .hyper.js**: 
- **Relevant information from devtools** _(CMD+ALT+I on macOS, CTRL+SHIFT+I elsewhere)_: 
- **The issue is reproducible in vanilla Hyper.app**: 

## Issue
Related to https://github.com/zeit/hyper/issues/2653

Emoji are not recognised by Hyper anymore

![gif](https://user-images.githubusercontent.com/234708/42200045-7a48136a-7e89-11e8-84c4-a1156567296b.gif)

**Comment by spencerwooo** [comment]
Hi, I'm also having issues with this. Do you have any updates?

**Comment by TheLetslook** [comment]
I have the same behavior on Hyper 2.1.0-canary.3, Windows 10 1809 build 17763.134

**Comment by spencerwooo** [comment]
![hyper_2018-12-16_22-51-33](https://user-images.githubusercontent.com/32114380/50055141-91ce3800-0185-11e9-8eba-b7832c5b30ef.png)

I don't know if this issue is being solved, but I recently updated to **Hyper 2.1.0-canary.5** and I can show **some emojis** on Windows. **But most emojis still can't be rendered correctly.**

My font settings are:

```json
// font family with optional fallbacks
fontFamily: '"Sarasa Mono SC", "Iosevka", "Segoe UI Emoji"'
```

(I use Iosevka as a fallback font simply for nerd font support.)

Also, even if I use Segoe UI Emoji as my main font, I still don't get emojis rendered correctly.

PS: The screenshot is from Windows PowerShell. Windows 10 version 1809.

**Comment by spences10** [comment]
Stale, closing, no one cares 😢
